docs/src/radiative_equilibrium.md

```diff
@@ -0,0 +1,103 @@
+
+
+# Radiative Equilibrium
+Radiative equilibrium is a useful framework for analyzing atmospheric temperature profiles with different atmosphere conditions. In radiative equilibrium, we only consider heat exchange from radiation.
+
+To run a radiative equilibrium simulation, create a yaml file (or use a default one) and run the following commands in your terminal (after instantiating the necessary packages):
+
+The yaml file should look something like this:
+`mysim.yml`
+```
+rad: "clearsky" 
+idealized_h2o: true 
+hyperdiff: "false" 
+config: "column" 
+initial_condition: "IsothermalProfile" 
+z_max: 70000
+z_elem: 70 
+dz_bottom: 100 
+dz_top: 10000 
+t_end: "654days" 
+dt: "3hours" 
+dt_save_to_sol: "30hours" 
+dt_save_to_disk: "100days" 
+prognostic_surface: "true" 
+job_id: "single_column_radiative_equilibrium_clearsky_prognostic_surface_temp"
+```
+
+Then run the simulation with the following command:
+```
+julia --project=examples examples/driver.jl --config_file path to mysim.yml
+```
+
+The configuration in the yaml file above can be changed to adjust to the kind of simulation desired. Options such as t\_end can be increased to the desired simulation time. Other arguments such as radiation type can be changed based on the options in `default_config 2.yml` and `default_config.yml`.
+
+After a simulation is ran the output files should be available in the output folder, where hdf5 files store all the simulation's data, and two mp4s should be produced with a vertical temperature profile and a vertical wind profile which change over time. If you want to quickly access data from the final day of the simulation simply add a -i to the command line input, like so:
+
+```
+julia -i --project=examples examples/driver.jl --config_file filepath
+```
+
+This will bring you into a julia environment where the current state of the simulation can be accessed. Run propertynames(Y) and propertynames(p) to see accesible values, some examples:
+
+![](assets/sample_-i_outputs.png)
+
+
+Here is a typical vertical temperature profile on the final day of a simulation (the dot is surface temperature):
+
+![](assets/example_temp_profile.png)
+
+Note: The surface temperature must also converge before the simulation reaches radiative equilibrium, here is a typical convergence of surface temperature plot:
+
+![](assets/surface_temp_convergence.png)
+
+
+# Greenhouse Gas Concentrations
+In `radiation.jl` one can change the concentrations of greenhouse gases in order to test the affects on the atmospheric temperature profile.
+
+To change the ozone concentration multiply the following variable the desired coefficent (line 89):
+
+![](assets/O3_sample_code.png)
+
+Then comparing the results between ozone simulations could look something like this:
+
+![](assets/O3_temp_profile.png)
+![](assets/O3_temp_difference.png)
+
+To change the CO2 concentration multiply the volume mixing ratio by the desired coefficient (line 100):
```

Comment:
```suggestion
To change the CO2 concentration, multiply the volume mixing ratio by the desired coefficient (line 100):
```
Same here, it would be better to link to the code instead of having a screen shot.